tests/Makefile.am

```diff
@@ -345,7 +345,12 @@ endif # HAVE_VALGRIND
 endif # ENABLE_DEFAULT_TESTS
 
 
+if ENABLE_SNMP
+TESTS +=  \
+	omsnmp_errmsg_no_params.sh
+endif # if ENABLE_SNMP
 if ENABLE_MAIL
+
```

Comment:
I guess the empty line should be in front of the new `if`?

Reply:
fixed